.mailmap

```diff
@@ -0,0 +1,5 @@
+Andrew Kofink <ajkofink@gmail.com> <akofink@redhat.com>
+Bernhard Hopfenmüller <hopfenmueller@atix.de> <bernhardh@atix.de>
+Christoffer Reijer <christoffer.reijer@basalt.se> <ephracis@gmail.com>
```

Comment:
@ephracis  please review which address you want as primary

Reply:
I would prefer the gmail address

Reply:
done, thanks!